pkg/tnf/handlers/podantiaffinity/podantiaffinity.json

```diff
@@ -0,0 +1,34 @@
+{
+  "identifier": {
+    "url": "http://test-network-function.com/tests/podantiaffinity",
+    "version": "v1.0.0"
+  },
+  "description": "This test checks cnf pod antiaffinity rule in cnf deployment.",
+  "testResult": 0,
+  "testTimeout": 10000000000,
+  "reelFirstStep": {
+    "execute": "oc get deployment {{.DEPLOYMENT_NAME}} -n {{.DEPLOYMENT_NAMESPACE}} -o json | jq -r ' if .spec.replicas > 1 then if .spec.template.spec.affinity.podAntiAffinity != null then \"OK\" else \"PODAAFAIL\" end else \"REPLICAFAIL\" end '\n",
+    "expect": [
+      "(?m)OK",
+      "(?m)PODAAFAIL",
+      "(?m)REPLICAFAIL"
+    ],
+    "timeout": 10000000000
+  },
+  "resultContexts": [
+    {
+      "pattern": "(?m)OK",
+      "defaultResult": 1
+    },
+    {
+      "pattern": "(?m)PODAAFAIL",
+      "defaultResult": 2
+    },
+    {
+      "pattern": "(?m)REPLICAFAIL",
+      "defaultResult": 2
```

Comment:
I think it's best practice "invented by me" to add a 
```
{
      "pattern": "(?m)[.*]",
      "defaultResult": 0
    }
```
in the end, this will catch everything else and return an ```error```

Reply:
"0" will be considered as tnf.ERROR, replica =1 is not error but  failure, right?
i think i can add one more pattern as you list above for  tnf.ERROR

Reply:
This is to catch anything unexpected, for example a failure in executing the command...

Reply:
@HamadiSE Shouldn't this behavior realized in code instead of json?

Reply:
@jun-rh That's also a solution. We should just adhere to the principal that the default return code is error.

Reply:
Yes